2016/proposals/vora-meet.md

```diff
@@ -190,20 +190,20 @@ The documentation will be finalized. Code will be beautified along with removal
-Being motivated to contribute to an open-source organization and having worked on Django for more than a year, I believe this project would be a perfect start to contribute to OSS. As a member of [Information Management Group](https://www.facebook.com/IMGIITRoorkee), I have worked on many django-based applications and have primarily worked on developing	Media Management Service and the Institute Placement Portal. The online competition portal that I developed for demonstration purpose can be found [here](https://github.com/meet-vora/euclid). This project seems very exciting to work on, given my background. Further, I have been able to use the SurveyMonkey API as well as Plotly during the discussion period and have well understood the idea. I have also realised the trouble one could get into, while using the SurveyMonkey API - querying with inclusion of custom variable *workshop_id* and OAuth, and have discussed the other details and issues [here](https://github.com/numfocus/gsoc/issues/98). Thus, I believe that I am well equipped with the skills, excitement and motivation to work on this project.  
+Being motivated to contribute to an open-source organization and having worked on Django for more than a year, I believe this project would be a perfect start to contribute to OSS. As a member of [Information Management Group](http://img.channeli.in/works/), I have worked on many Django-based applications and have primarily worked on developing	Media Management Service and the Institute Placement Portal. The online competition portal that I developed for demonstration purpose can be found [here](https://github.com/meet-vora/euclid). This project seems very exciting to work on, given my background. Further, I have been able to use the SurveyMonkey API as well as Plotly during the discussion period and have well understood the idea. I have also realised the trouble one could get into, while using the SurveyMonkey API - querying with inclusion of custom variable *workshop_id* and OAuth, and have discussed the other details and issues [here](https://github.com/numfocus/gsoc/issues/98). Thus, I believe that I am well equipped with the skills, excitement and motivation to work on this project.  
 
 ## Other Details
 #### Summer Schedule
-I have no other commitments scheduled for the summer and would devote atleast 50 hours a week, and more if required.
-The ongoing semester will be completed by 10th of May. The next semester is scheduled to begin on 20th of July, 2016. 
+I have no other commitments scheduled for the summer and would devote at least 40 hours a week, and more if required.
+The ongoing semester will be completed by 6th of May. The next semester is scheduled to begin on 20th of July, 2016. 
 
 #### Contact
 Email: [meetvora@outlook.com](mailto:meetvora@outlook.com) / [v.h.c.meetirth@gmail.com](mailto:v.h.c.meetirth@gmail.com)  
-Mobile: +91 839 487 7088
\ No newline at end of file
+Mobile: +91 839 487 7088 / +91 702 198 4036
```

Comment:
Uh, maybe it's my fault, but somehow this ↑ comment was placed under wrong line…